### ISSUE TITLE
Allow the user to specify where the Ember root directory is

### ISSUE THREAD

**Issue by discopatrick** [Turbo87/intellij-emberjs#188]
We are having a problem using this plugin, because our Ember project (and thus our `node_modules` directory) is not in the project root. This is because our project consists of various sub-projects, including a Django REST backend (in a `backend` dir), as well as the Ember project (in a `frontend` dir).

When we try to run the Ember Serve run configuration, we get the error `Error running 'MyProject': Cannot run program "/home/me/dev/myproject/node_modules/.bin/ember" (in directory "/home/me/dev/myproject"): error=2, No such file or directory`

The correct path to ember should be: /home/me/dev/myproject/**frontend**/node_modules/.bin/ember.

Can we add a setting to specify the Ember root dir?

**Comment by IgorKvasn** [comment]
I've got a similar project setup as you:
```<root>
 - submodule1
 - submodule2
 - ember module
```
and this plugin runs without a problem.
I am not sure how to help you though :(

**Comment by makepanic** [comment]
@discopatrick `<frontend>` is a different module in your project right?
Does this only affect the ember serve run configuration?

**Comment by discopatrick** [comment]
Yes, it's just another folder in the project. And yes, it only seems to affect the Ember side of things. For my Django backend, I can specify the Working Directory to be `backend`.

Hmm, is it possible to choose the root folder for the Ember plugin? Maybe I missed this...

**Comment by makepanic** [comment]
I was asking because we have an issue for adding support of run configuration per module: https://github.com/Turbo87/intellij-emberjs/issues/181
This wouldn't fix your issue though. So maybe it would be better to be able to set the working directory instead of a module.

**Comment by RavenHursT** [comment]
I'm seeing a similar problem w/ resolution of `import` statements throughout my project.  I have a similar structure to @discopatrick :

```
/dev
  /my-project
    /app
    /config
    etc etc
```
CMD-clicking on anything like `import bar from 'my-project/foo/bar'` results in the message "Cannot find a declaration to go to"

Any help w/ this would be GREAT!

**Comment by JonnoFTW** [comment]
Is this issue going to be fixed?

**Comment by Turbo87** [comment]
PRs welcome 😉
